submissions/termflow/termflow.py

```diff
@@ -0,0 +1,236 @@
+import json
+import shlex
+import subprocess
+from pathlib import Path
+from typing import List, Dict
+from rich.console import Console
```

Comment:
please include a `requirements.txt` file including all the dependencies that are needed to run this project.